app/services/settings/general/upsert.rb

```diff
@@ -0,0 +1,34 @@
+module Settings
+  class General
+    class Upsert
```

Comment:
Since `Settings::Authentication::User` is a module and it doesn't look like we're instantiating this, should this one be a module, too, for consistency?

```suggestion
    module Upsert
```

Reply:
Yes, good catch. 👍  I had changed this one back and forth a few times must have forgotten to change it back the last time. Thanks!